### PR TITLE
Add singletile option to layer's menu

### PR DESCRIPTION
**Pull request by jusabatier**
This adds an "Activate SingleTile" option in the Action menu of WMS layers.

It allow to set labels layers as singleTile, so they don't cut at tiles side.

### CODE REVIEW

**Comment by fvanderbiest** [comment]
Looks good, but is it persisted and restored to/from WMC ?

**Comment by jusabatier** [comment]
Tried just now and yes, the option is persisted in the generated WMC (Save map) and also restored from a loaded WMC.

**Comment by jusabatier** [comment]
Can you merge this PR for 15.12/16.12 ?

File: mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js [hunk 1]
```diff
@@ -636,7 +636,8 @@ GEOR.managelayers = (function() {
                 layerRecord.hasEquivalentWCS() : false,
             isVector = layer instanceof OpenLayers.Layer.Vector,
             isBaseLayer = layerRecord.get("opaque") || 
-                layer.transitionEffect === "resize";
+                layer.transitionEffect === "resize",
+            isSingleTile = layer.options.singleTile;
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js]: singleTile is a layer property, see  https://github.com/georchestra/openlayers/blob/master/lib/OpenLayers/Layer/Grid.js#L68-L75

File: mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js [hunk 2]
```diff
@@ -842,6 +843,19 @@ GEOR.managelayers = (function() {
                 }
             });
         }
+        
+        if (isWMS) {
+            menuItems.push({
+                text: isSingleTile ? 
+                    tr("Deactivate SingleTile") : tr("Activate SingleTile"),
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js]: Translations are missing too...

Comment (jusabatier) [inline mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js]: Done

Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js]: Hmm, not exactly: missing items + incorrect position in translation file.

Comment (jusabatier) [inline mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js]: Right I forget "Activate", should be OK now

File: mapfishapp/src/main/webapp/app/js/GEOR_Lang/fr.js [hunk 3]
```diff
@@ -209,6 +209,8 @@ OpenLayers.Lang.fr = OpenLayers.Util.extend(OpenLayers.Lang.fr, {
     "About this layer": "A propos de cette couche",
     "Set as overlay": "Passer en calque",
     "Set as baselayer": "Passer en couche de fond",
+    "Deactivate SingleTile" : "Desactiver SingleTile",
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_Lang/fr.js]: Désactiver

File: mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js [hunk 4]
```diff
@@ -842,6 +843,19 @@ GEOR.managelayers = (function() {
...
+                handler: function() {
+                    layerRecord.get('layer').addOptions({singleTile: !isSingleTile});
+                    layerRecord.get('layer').redraw(true);
+                    isSingleTile = !isSingleTile;
+                    this.setText(isSingleTile ? tr("Deactivate SingleTile") : tr("Activate SingleTile"));
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js]: Je me demande s'il ne serait pas plus pertinent de mettre une checkbox ici, avec comme intitulé "mode tuilé".

Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js]: La case serait alors cochée par défaut.

Comment (jusabatier) [inline mapfishapp/src/main/webapp/app/js/GEOR_managelayers.js]: C'est fait

**Comment by fvanderbiest** [comment]
Did you check this works as expected ?
You seem to edit using the only editor, which makes me think you did not.

Merging a PR targeting a stable release must be done with confidence.
Thanks for your contribution on this.

**Comment by jusabatier** [comment]
Before modify the PR via online editor, I modify my test instance in debug mode and check the modifications are working.

File: mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js [hunk 5]
```diff
@@ -227,6 +227,7 @@ OpenLayers.Lang.de = OpenLayers.Util.extend(OpenLayers.Lang.de, {
     "Download data": "Daten downloaden",
     "Choose a style": "Stil",
     "Modify format": "Format ändern",
+    "Tiled mode" : "Gefliester Modus",
```
Comment (fvanderbiest) [inline mapfishapp/src/main/webapp/app/js/GEOR_Lang/de.js]: Gekachelt Modus

**Comment by fvanderbiest** [comment]
Now this looks good. Thanks !

**Comment by fvanderbiest** [comment]
Cascade merged from 15.12 into 16.12 and master.

**Pull request merged**